features/FEATURE_BLE/ble/gap/Gap.h

```diff
@@ -492,6 +492,41 @@ class Gap {
         {
         }
 
+        /**
+         * Function invoked when the connections changes the ATT_MTU which controls
+         * the maximum size of an attribute that can be read in a single L2CAP packet
+         * which might be fragmented across multiple packets.
+         *
+         * @param connectionHandle The handle of the connection that changed the size.
+         * @param attMtuSize
+         */
+        virtual void onAttMtuChange(
```

Comment:
I wonder if we shouldn't have an event in `GattServer` and another in `GattClient` rather than one in GAP. I think that is not the place where this event is expected even if I understand why it is there (there is not GattCommon). 
@donatieng What do you think ?

Reply:
Unfortunately there is no distinction of server and client for the Cordio generated event - I wouldn't know which event to call (client or server), it's the only event that is shared between client and server.